### ISSUE TITLE
Vagrant setup incomplete

### ISSUE THREAD

**Issue by yombo** [yombo/yombo-gateway#13]
When using provision.sh, it doesn't seem to complete the install. Looks like pyenv isn't fully configured as python modes are not installed.

**Comment by yombo** [comment]
It appears the setup is being blocked by a prompt from keyboard-configuration and grub-c packages.

Making changes to the setup and install scripts to mark these packages on hold during apt-get upgrade.
